lib/stencil-push.utils.js

```diff
@@ -281,7 +285,8 @@ utils.promptUserWhetherToApplyTheme = (options, callback) => {
         Inquirer.prompt(questions, answers => {
             callback(null, Object.assign({}, options, { applyTheme: answers.applyTheme }));
         });
-    };
+    }
+    ;
```

Comment:
Can you please collapse the semi-colons and braces back to one line in this file.

Reply:
I'll do you one better and remove the semi-colon. My editor tried to auto-fix this styling, but it was confused (as it should be) because this semi-colon isn't supposed to exist. It's trying to terminate the `else` block.